src/engine/game/gui/ingame/equip.h

```diff
@@ -65,6 +71,53 @@ class Equipment : public GameGUI {
 
     std::shared_ptr<graphics::Texture> getItemFrameTexture(int stackSize) const;
     std::shared_ptr<graphics::Texture> getEmptySlotIcon(Slot slot) const;
+
+    struct Binding {
+        Label *lblCantEquip { nullptr };
+        Label *lblAttackInfo { nullptr };
+        Label *lblToHitR { nullptr };
+        Label *lblPortBord { nullptr };
+        Label *lblPortrait { nullptr };
+        ListBox *lbItems { nullptr };
+        Label *lblDefInfo { nullptr };
+        unordered_map<Equipment::Slot, Label *> lblInv;
+        unordered_map<Equipment::Slot, Button *> btnInv;
+        Label *lblAtkL { nullptr };
+        Label *lblAtkR { nullptr };
+        Label *lblDef { nullptr };
+        Label *lblTitle { nullptr };
+        Label *lblDamage { nullptr };
+        Label *lblToHitL { nullptr };
+        Label *lblToHit { nullptr };
+        Label *lblSlotName { nullptr };
+        Button *btnBack { nullptr };
+        Button *btnEquip{ nullptr };
+        ListBox *lbDesc { nullptr };
+
+        // KOTOR only
+        Label *lblVitality { nullptr };
+        Label *lblTxtBar { nullptr };
+        Button *btnCharLeft { nullptr };
+        Button *btnCharRight { nullptr };
+        Label *lblSelectTitle { nullptr };
+        Button *btnChange1 { nullptr };
+        Button *btnChange2 { nullptr };
+        // End KOTOR only
+
+        // TSL only
+        Label *lblBack1 { nullptr };
+        Label *lblDefBack { nullptr };
+        Label *lblBar[5];
+        Label *lblAttackMod { nullptr };
+        Label *lblDamText { nullptr };
+        Button *btnSwapWeapons { nullptr };
+        Button *btnPrevNpc { nullptr };
+        Button *btnNextNpc { nullptr };
+        Label *lblDefText { nullptr };
+        // End TSL only
+    };
+    Binding binding;
```

Comment:
Member variables must be prefixed with an underscore, as per the code style (that is only for classes, in structs they must not).

Reply:
Declaration could be shortened just a bit:
```
struct Binding {
    ...
} _binding;
```